### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by kevmoo**
@alan-knight Mind a GitHub pull request?

### CODE REVIEW

**Comment by alan-knight** [comment]
Can you update the changelog? Also I can't see what's changed because of all the reformatting. Either don't reformat, or at least don't reformat files you're not otherwise changing, or do something that's no change except formatting. Reformatting very large generated files just makes for a lot of noise.

**Comment by kevmoo** [comment]
@alan-knight That's actually the nice thing about github. You can look at Commits above and see the change for each commit - without doing 5 separate reviews.

Will update changelog.

**Comment by kevmoo** [comment]
@alan-knight PTAL

**Comment by alan-knight** [comment]
There's some ugliness in the formatting, but not actually as bad as I expected, because I have nice short variable names on the tests. It's tolerable. But I do need that deleted statement.

**Comment by kevmoo** [comment]
If the tests pass with and without the statement, shouldn't we update the test so it's needed?

Also: where is that extra semi-colon

File: lib/intl_standalone.dart [hunk 1]
```diff
@@ -63,7 +63,8 @@ RegExp _appleDefaultsRegex = new RegExp(r'((\w\w)_\w+)');
 String _checkEnvironmentVariable() {
   try {
     return Platform.environment['LANG'];
-  } catch (e) {};
+  } catch (e) {}
+  ;
```
Comment (alan-knight) [inline lib/intl_standalone.dart]: Standalone semicolon is here.

File: test/message_extraction/part_of_sample_with_messages.dart [hunk 2]
```diff
@@ -13,20 +13,19 @@ class Person {
...
+  method() => Intl.message("This comes from a method",
+      name: 'method', desc: 'This is a method with a '
+      'long description which spans '
+      'multiple lines.');
 
   // A non-lambda, i.e. not using => syntax, and with an additional statement
   // before the Intl.message call.
   nonLambda() {
     // TODO(alanknight): I'm really not sure that this shouldn't be disallowed.
-    var x = 'something';
```
Comment (alan-knight) [inline test/message_extraction/part_of_sample_with_messages.dart]: This line.

Comment (alan-knight) [inline test/message_extraction/part_of_sample_with_messages.dart]: I'm not sure how to write the test in such a way that it fails if this is removed.

**Pull request merged**